sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsAutoConfigurationTests.java

```diff
@@ -149,6 +139,18 @@ void autoconfigurationEnabledAndContextSuccessWithPremiumTier(String pricingTier
             });
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = { "basic", "standard", "premium" })
+    void doesnotHaveBeanOfAzureServiceBusJmsPropertiesBeanPostProcessor(String pricingTier) {
+        this.contextRunner
+            .withPropertyValues(
+                "spring.jms.servicebus.pricing-tier=" + pricingTier,
+                "spring.jms.servicebus.connection-string=" + CONNECTION_STRING)
+            .run(context -> {
+                assertThat(context).doesNotHaveBean(AzureServiceBusJmsPropertiesBeanPostProcessor.class);
+            });
+    }
+
```

Comment:
What about the case of having a post processor?

Reply:
And should we have a test case to make sure the post processor is working?

Reply:
yes, it is a good idea.